services/workflow/workflow-service/src/main/java/com/milesight/beaveriot/rule/components/serviceinvocation/ServiceInvocationComponent.java

```diff
@@ -35,6 +36,7 @@ public class ServiceInvocationComponent implements ProcessorNode<Exchange> {
 
     @Override
     public void processor(Exchange exchange) {
+        Map<String,Object> serviceParams = JsonHelper.fromJSON(serviceInvocationSetting.get("serviceParams").toString(), Map.class);
```

Comment:
Will there be an empty case for serviceParams?

Reply:
fixed